app/controllers/employees_controller.rb

```diff
@@ -50,8 +40,6 @@ def create
     end
   end
 
-  # PATCH/PUT /employees/1
-  # PATCH/PUT /employees/1.json
   def update
     params[:employee][:skill_ids] ||=[]
```

Comment:
Try not to modify your params hash. It might be better to store this in some sort of variable by something like the following:

@skill_ids = params[:employee][:skill_ids].nil? [] : params[:employee][:skill_ids] 

or 

@skill_ids = Array(params[:employee][:skill_ids])